src/org/omegat/gui/main/MainWindow.java

```diff
@@ -461,11 +472,38 @@ public void displayErrorRB(final Throwable ex, final String errorKey, final Obje
             }
 
             statusLabel.setText(msg);
-            String fulltext = msg;
+
+            JPanel pane = new JPanel();
+            pane.setLayout(new BoxLayout(pane, BoxLayout.PAGE_AXIS));
+            pane.setSize(new Dimension(900, 400));
+
+            JLabel jlabel = new JLabel(msg);
+            jlabel.setAlignmentX(LEFT_ALIGNMENT);
+            pane.add(jlabel);
+
             if (ex != null) {
-                fulltext += "\n" + ex.toString();
+                pane.add(Box.createRigidArea(new Dimension(0, 5)));
+                JTextArea message = new JTextArea();
+                message.setText(ex.getLocalizedMessage());
+                message.setLineWrap(true);
+                message.setEditable(false);
+                JScrollPane jScrollPane = new JScrollPane(message,
+                        ScrollPaneConstants.VERTICAL_SCROLLBAR_ALWAYS,
+                        ScrollPaneConstants.HORIZONTAL_SCROLLBAR_NEVER);
+                jScrollPane.setAlignmentX(LEFT_ALIGNMENT);
+                jScrollPane.setPreferredSize(new Dimension(800, 200));
+                jScrollPane.getVerticalScrollBar().setValue(0);
+                pane.add(jScrollPane);
+                pane.setBorder(BorderFactory.createEmptyBorder(10, 10, 10, 10));
+                JButton jbutton = new JButton(OStrings.getString("TF_ERROR_COPY_CLIPBOARD"));
+                // Copy to clipboard action
+                jbutton.addActionListener(l -> Toolkit.getDefaultToolkit().getSystemClipboard()
+                        .setContents(new StringSelection(ex.getLocalizedMessage()), null));
```

Comment:
it may be greater if it also copy platform information (omegat version, java version, os) to clipboard.
see https://sourceforge.net/p/omegat/feature-requests/1683/

Reply:
Excellent idea, I updated the PR so the Support Info are included at the end of the error message, like:

```
C:/Users/briac/Downloads/error-wrap/source/xml/one of those errors where the text is wrapped on Linux but not on Windows/a_folder_with_a_somewhat_long_name/this is a test to see if the text is correctly wrapped/This is a very Long filename I wonder if it will be displayed correctly.xml
net.sf.okapi.common.exceptions.OkapiIOException: Parsing error.
The element type "bad" must be terminated by the matching end-tag "</bad>".
---
Version: OmegaT-5.8.0_0_dev
Platform: Windows 10 10.0
Java: 11.0.1 amd64
Memory: 152MiB total / 48MiB free / 1992MiB max
```